shared/menubar/remote-proxy.desktop.tsx

```diff
@@ -136,7 +136,7 @@ const mapStateToProps = (state: Container.TypedState) => ({
   showingDiskSpaceBanner: state.fs.overallSyncStatus.showingBanner,
   userInfo: state.users.infoMap,
   username: state.config.username,
-  widgetBadge: state.notifications.get('widgetBadge') || 'regular',
+  widgetBadge: state.notifications.widgetBadge || 'regular',
```

Comment:
Is the `|| 'regular'` part necessary? It's populated from the initial state right?

Reply:
i agree it can be removed, updated